Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1187,7 +1187,7 @@ namespace Babylon
         const auto texture = info[0].As<Napi::External<TextureData>>().Data();
         const auto value = info[1].As<Napi::Number>().Uint32Value();
 
-        texture->AnisotropicLevel = value;
```

Comment:
I think we’d want narrow rather than narrow_cast. narrow_cast just does a static_cast, but narrow does runtime checks for data loss. @bghgary thoughts?

Reply:
The only difference between narrow and narrow_cast is whether it throws or not if the value doesn't fit. My opinion is that since this is an implementation detail of the engine, we don't need a runtime check. Is there something that asserts?

Also, we should try to be more consistent in our usage of static_cast vs gsl casts.

Reply:
@bghgary do you mean runtime asserts? Ultimately that would just end up with a runtime exception anyway, right? We can't do compile time asserts since we don't know these values at compile time.

Reply:
Yes, but I mean debug asserts, so that it doesn't impact perf.

Reply:
🤷🏼‍♂️

Reply:
Using static_cast for now. replacing some with gsl::narrow_cast will be done in another PR.